features/data.feature

```diff
@@ -59,6 +59,20 @@ Feature: Data
     And I should see "Jack" in "_site/index.html"
     And I should see "Leon" in "_site/index.html"
 
+  Scenario: autoload *.tsv files in _data directory
+    Given I have a _data directory
+    And I have a "_data/members.tsv" file with content:
+      """
+      name  age
+      Jack  28
+      Leon  34
```

Comment:
No, that's not required. I pulled your branch locally to check. Its seems you delimited by `2 \s` (2 `space` characters instead of a `tab` character